### DISCUSSION TITLE
How to move the cusror to next line when an option is selected

### DISCUSSION THREAD

**Discussion by pdhruv93** [JedWatson/react-select#5788] [Q&A]
I have a multi-select component and want the cursor on the new/next line whenever the option is selected so that the new option is always selected on a new line.

**Comment by manjushsh** [comment]
Hi @pdhruv93, If I am understanding you requirements correctly, I think there is no props for this.

Here is [working example implemented using custom CSS.](https://codesandbox.io/s/affectionate-wood-wrt236?file=/src/App.js)

**Reply by pdhruv93** [reply]
Thanks @manjushsh, this was my requirement. So there is no prop for this, need to do with styling
